src/MinimalDelegation.sol

```diff
@@ -140,24 +139,19 @@ contract MinimalDelegation is
             }
         }
 
-        (bytes32 keyHash, bytes memory signature, bytes memory hookData) =
-            abi.decode(wrappedSignature, (bytes32, bytes, bytes));
+        (bytes32 keyHash, bytes calldata signature, bytes memory hookData) =
```

Comment:
this is odd to me that we return part of this in calldata and part in memory?

Reply:
Fixed!